core/src/main/java/org/testcontainers/images/builder/Transferable.java

```diff
@@ -5,12 +5,18 @@
 import org.apache.commons.io.IOUtils;
 
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.zip.Checksum;
 
 public interface Transferable {
 
     int DEFAULT_FILE_MODE = 0100644;
     int DEFAULT_DIR_MODE = 040755;
 
+    static Transferable of(String string) {
+        return of(string.getBytes(StandardCharsets.UTF_8));
+    }
```

Comment:
This is the other change leading to `japicmp` error. I am not sure what is the desired pattern to avoid this.

Reply:
wait, what? Added static method makes `japicmp` report it as binary incompatible change? o_O What does it say exactly?

Reply:
See the other comment, it fails because of `METHOD_NEW_DEFAULT` rule violation. But I can't say if this is a FP in this case.